torchbiggraph/train.py

```diff
@@ -747,6 +747,18 @@ def log_status(msg, always=False):
         sync.barrier()
         dlog("All parts of the checkpoint have been written")
 
+        if config.checkpoint_interval is not None:
+            # If it is the right interval for saving, we copy all current
+            # checkpoints into it's own separate folder
+            is_save_interval = epoch_idx % config.checkpoint_interval == 0
+            is_last_edge_path = \
+                (edge_path_idx + 1) == iteration_manager.num_edge_paths
+            is_last_edge_chunk = \
+                (edge_chunk_idx + 1) == iteration_manager.num_edge_chunks
```

Comment:
It maybe clearer to add a `is_last_iteration_of_epoch` method (or a better name) to the `iteration_manager` and hide this code in there.